### PR TITLE
Show Version info

### LINKED ISSUE #45: Show information about HNAP in index.html and info.html
The Pluto offers an info page on the Mass-Storage device and on its webserver. It shows basic information about the current versions of the various parts of the system (firmware, linux, u-boot, etc.), a short getting started and an update notifier.

Basically, it comes from `plutosdr-fw/buildroot/board/pluto/index.html` and is altered in various steps during compile time (information about the compiler is filled in) of the firmware and the start of the system (current settings are filled in).
Currently it is a static page, which checks for a new Pluto firmware image version using Javascript.
We could do a similar thing and check for a new HNAP firmware version.
Additionally, we could update the information on this page dynamically using javascript, so we can have a nice dashboard about the current state of the device.

We could put HNAP related information on this page:

- [x] HNAP logo
- [x] HNAP firmware version
- [ ] HNAP settings
- [x] HNAP firmware update notification
- [ ] HNAP statistics and current state of the device (connectivity, statistics), ...

### LINKED ISSUE #48: Show HNAP Version information in MOTD and on the infopage
To place the info about the currently used HNAP4PlutoSDR Firmware inforrmation more prominent, we should just put it into the motd and the infopage (accessible at http://192.168.2.1 or in the MSD).

### PR DESCRIPTION
**Pull request by dl1com**
- [x] Show Version info in MOTD
image
- [x] Show Version info in Webpage
image

- [x] Redirect version checker to HNAP releases
- [x] Remove obsolete files in buildroot/board/pluto/msd/img folder

- [x] Show warning in MOTD if root password is still set to default value

### CODE REVIEW

File: overlay/buildroot/index.html [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+<a class="button" href="#support">Help and Support</a>
+<a class="button" href="LICENSE.html">License</a>
+</nav>
+<hr>
+<section id="firmware">
+<a class="anchor" href="#top">Back to top</a>
+<h3>Firmware</h3>
+<div id="versionsection" class="download">
+<p><strong>Status of the HNAP4PlutoSDR firmware:</strong></p>
+<p><span id="versiontest">Need javascript to check (sorry)</span></p>
```
Comment (lukasostendorf) [inline overlay/buildroot/index.html:52]: The script that replaces this message encounters an error (see below). My Browser only displays the "Need Javascript to check" message.

File: overlay/buildroot/msd/img/version.js [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+		var uAgent = navigator.userAgent.toLowerCase();
+		//uAgent = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.90 Safari/537.36".toLowerCase();
+		var test = uAgent.match(/mac os x 10_[0-9]*/);
+		if (test) {
+			mac = "darwin-" + test[0].match(/10_[0-9]*/)[0].replace("_", ".");
+		}
+		document.getElementById('prompt0').innerHTML = "adi-mm:tests analogdevices$";
+		document.getElementById('prompt1').innerHTML = "adi-mm:tests analogdevices$";
+	} else {
+		document.getElementById('hidehorndis').style.display = "none";
```
Comment (lukasostendorf) [inline overlay/buildroot/msd/img/version.js:61]: My Browser (Firefox 82) encounters the following error:
```
Uncaught TypeError: document.getElementById(...) is null
    GetDriverurl http://192.168.2.1/img/version.js:46
    CheckFrmVersion http://192.168.2.1/img/version.js:218
```

**Review by lukasostendorf — COMMENTED** [review]
Build process works fine.
When checking the Pluto website, javascript execution throws an error and the current HNAP apparently cannot be fetched.

File: overlay/buildroot/msd/img/version.js (hunk 2 above)
Comment (dl1com) [reply overlay/buildroot/msd/img/version.js:61]: Fixed this by re-adding the section about libiio to the html file (makes sense as well, as we currently still support iio operation as well)

**Review by lukasostendorf — APPROVED** [review]

**Pull request merged**